### ISSUE TITLE
Support UTF8

### ISSUE THREAD

**Issue by winneze1** [CastawayLabs/dart-escpos#6]
Hi does this package support utf8? I really need it for my project.

**Comment by matejkramny** [comment]
Support really depends on the printer, however they use preset codetables which you need to use to achieve whatever characters need to be displayed.

There are workarounds, such as generating an image and printing that image. That's f.e. if you want to use a custom font, print emojis or other pictograms

**Comment by winneze1** [comment]
Yes currently i was using the workarounds way by generating an image. But that make my server to generate an image. Just asking if there is a future plan on implement this circumstance.
